ironfish-cli/src/commands/chain/download.ts

```diff
@@ -134,6 +134,23 @@ export default class Download extends IronfishCommand {
       await fsAsync.mkdir(this.sdk.config.tempDir, { recursive: true })
       snapshotPath = flags.outputPath || path.join(this.sdk.config.tempDir, manifest.file_name)
 
+      // Check if we can resume a previous download, and update the downloaded size if so
+      const headResult = await axios.head(snapshotUrl)
+      const canAcceptRanges =
+        (headResult.headers as Record<string, string>)['accept-ranges'] === 'bytes'
```

Comment:
Nit: you can in theory send a `Range` request even if the server does not understand it. If it supports it, it will respond with a [206 Partial Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/206), otherwise it will respond with a 200, see the [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range):

> The server can also ignore the Range header and return the whole document with a [200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200) status code

Therefore this HEAD request should not be necessary and can be avoided to minimize the number of roundtrips.

If you really want to test for server support, then I'd invert the order here: first check if the local snapshot exists (cheap), then ask the server (expensive).